### PR TITLE
Cleanup Connectors update guides

### PR DESCRIPTION
**Pull request by akeller**
## Description

Reversed the list of Connectors update guides. Removed extraneous tab import statements.

The only time tabs are used (and therefore, the import statements are needed) are on the first Connectors update guide - 0.10 to 0.20.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.1/guides/update-guide/connectors/introduction.md
Comment (christinaausley) [inline versioned_docs/version-8.1/guides/update-guide/connectors/introduction.md:3]: 👏 For this adjustment.

File: versioned_docs/version-8.2/guides/update-guide/connectors/030-to-040.md
Comment (christinaausley) [inline versioned_docs/version-8.2/guides/update-guide/connectors/030-to-040.md:7]: Similar to the Guides content, do you also feel Connectors need a review of skill level moving forward, or you're fairly confident with all of these as-is? Would you like to see time estimates as well (or is this solely something we want to keep within Guides for now?

**Review by christinaausley — APPROVED** [review]
This was tedious work but nice efforts here 👍 Going to wait on @sbuettner and @bastiankoerber for final review to merge. But if one of you gets to this first, feel free to squash and merge if approved on your end.

**Comment by akeller** [comment]
Bummer, it looks like this has some merge conflicts now. If I get back to this today, I'll clean it up.

**Review by christinaausley — APPROVED** [review]
I think I got those conflicts resolved for you @akeller, but please double check.

File: versioned_docs/version-8.2/guides/update-guide/connectors/030-to-040.md
Comment (akeller) [reply versioned_docs/version-8.2/guides/update-guide/connectors/030-to-040.md:7]: I would entirely remove the skill levels and time estimates from the `/update-guide/` section. We should push back on anything that looks lengthy or challenging, or if we have time to run through the actual guides (including update guides), confirm that they are straightforward.

Comment (akeller) [reply versioned_docs/version-8.2/guides/update-guide/connectors/030-to-040.md:7]: I realize this contradicts what I said in https://github.com/camunda/camunda-platform-docs/pull/2337.

Unless there are significant changes from version to version in an update guide, labeling each of them individually feels noisy.

**Review by christinaausley — DISMISSED** [review]

**Comment by akeller** [comment]
@sbuettner, have a look at this PR. The Connectors update guides do not need the import statement unless they include tabs in the page content. 0.1 to 0.2 is the only page that needs these.

Does your team have a template somewhere that needs to be updated? In my latest commit, I removed the import statements from 0.10 to 0.11.

**Comment by sbuettner** [comment]
Thank you @akeller. Not that I am aware of. And if so, we could remove them at a later point in time.

**Comment by akeller** [comment]
Can I get a 👍 from a reviewer? 😄

**Review by christinaausley — APPROVED** [review]

**Pull request merged**
